### PR TITLE
Add support for OCaml 5.00

### PR DESCRIPTION
**Pull request by kit-ty-kate**

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
Thanks! If you have the time, could you make a release if possible?

**Comment by pmetzger** [comment]
@kit-ty-kate Given that you're responsible for keeping these sorts of things working as OCaml releases come out, would you prefer to simply have access to the OCaml Community repositories yourself?

File: lambda-term.opam [hunk 1]
```diff
@@ -10,7 +10,7 @@ build: [
   ["dune" "runtest" "-p" name "-j" jobs] {with-test}
 ]
 depends: [
-  "ocaml" {>= "4.02.3"}
+  "ocaml" {>= "4.03.0"}
```
Comment (kit-ty-kate) [inline lambda-term.opam:13]:
```suggestion
  "ocaml" {>= "4.08.0"}
```
I just realized `Format.stag` and so on were added in OCaml 4.08 so this needs to change as well

**Comment by kit-ty-kate** [comment]
> Given that you're responsible for keeping these sorts of things working as OCaml releases come out, would you prefer to simply have access to the OCaml Community repositories yourself?

Sure, if you don’t have the time I can do it if given the rights. Though I’m already responsible for far too many things so if there is another solution it might be preferable.

**Comment by pmetzger** [comment]
I think it might actually save you time by allowing you to avoid needless interactions for obvious fixes. I'll add you shortly.

File: lambda-term.opam (hunk 1 above)
Comment (pmetzger) [reply lambda-term.opam:13]: You should now have access as soon as you accept it (check your email.)

**Pull request merged**